src/components/RegistrationButton.astro

```diff
@@ -1,17 +1,29 @@
 ---
-const { color } = Astro.props;
+export type Props = {
+	variant?: "black" | "red";
+};
+const { variant = "black" } = Astro.props;
 ---
 
 <a
-  href="https://lu.ma/event/evt-pYYlW3M6j0e8eAB"
-  class="button"
-  data-luma-action="checkout"
-  data-luma-event-id="evt-pYYlW3M6j0e8eAB"
-  style={color ? `--button-bg-color: ${color}` : null}
+	href="https://lu.ma/event/evt-pYYlW3M6j0e8eAB"
+	class="button"
+	data-variant={variant}
+	data-luma-action="checkout"
+	data-luma-event-id="evt-pYYlW3M6j0e8eAB"
 >
-  <slot />
+	<slot />
 </a>
 
-<style lang="scss"></style>
+<style lang="scss">
+	a[data-variant="red"] {
+		--button-bg-color: var(--accent-constant-red);
 
-<script id="luma-checkout" src="https://embed.lu.ma/checkout-button.js"></script>
+		&:where(:hover) {
+			--button-bg-color: var(--accent-constant-red-hover);
+		}
+	}
+</style>
+
+<script id="luma-checkout" src="https://embed.lu.ma/checkout-button.js"
+></script>
```

Comment:
Heads up, this line will likely conflict with #25, where I moved this script to the `<head>`

Reply:
Good catch! I undid the formatting changes so hopefully should be good.